### ISSUE TITLE
Open Footprint

### ISSUE THREAD

**Issue by johnantoni** [MozillaFestival/open-leaders-6#39]
Project Lead: johnantoni

Mentor: Solange TUYISENGE (to be updated)

Welcome to OL6, Cohort B! This issue will be used to track your project and progress during the program. Please use this checklist over the next few weeks as you start Open Leadership Training :tada:.

***
Before Week 1 (Sept 11): Your first mentorship call
- [ ] Complete the OLF self-assessment ([online](https://mzl.la/olf-online-assessment), [printable](https://mzl.la/olf-self-assessment)). If you're a group, each teammate should complete this assessment individually. This is here to help you set your own personal goals during the program. No need to share your results, but be ready to share your thoughts with your mentor.
- [ ] Make sure you know when and how you'll be meeting with your mentor.

Before Week 2 (Sept 18): First Cohort Call (Open by Design)
- [ ] Complete Assignment ['Write Your Vision or Mission Statement'](https://mozilla.github.io/open-leadership-training-series/articles/introduction-to-open-leadership/stating-your-project-vision/). Leave a comment on this issue with your draft vision statement & be ready to share this on the call.
- [ ] Watch ['Mozilla & Working Open'](https://youtu.be/quKdaqlR_9w).
- [ ] Check the [Syllabus](https://mzl.la/ol6-syllabus) for notes and connection info for all the cohort calls.

Before Week 3 (Sept 25): Mentorship call
- [ ] Look up two other projects and comment on their issues with feedback on their vision statement.
- [ ] Complete your Open Canvas ([instructions](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/develop-an-open-project-strategy-with-open-canvas/), [canvas](https://goo.gl/to6PYn)). Comment on this issue with a link to your canvas.
- [ ] [Start your Roadmap](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/start-your-project-roadmap/). Comment on this issue with your draft Roadmap.

Before Week 4 (Oct 2): Cohort Call (Build for Understanding)
- [ ] Look up two other projects and comment on their issues with feedback on their open canvas.
- [ ] [Pick an open license](https://mozilla.github.io/open-leadership-training-series/articles/get-your-project-online/sharing-your-work-in-the-open/) for the work you're doing during the program.
- [ ] Use your canvas to [start writing a README](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/write-a-great-project-readme/), or landing page, for your project. Link to your README in a comment on this issue.

This issue is here to help you keep track of work during the first month of the program. Please refer to the [OL6 Syllabus](https://mzl.la/ol6-syllabus) for more detailed weekly notes and assignments past week 4.

**Comment by johnantoni** [comment]
Vision Statement:

> I'm working with a small community of freelancers to further internet literacy skills by creating a tool to map your social impact on the web so students and individuals can stay safe while on the world wide web.

Repo: https://github.com/johnantoni/open-footprint

**Comment by amchagas** [comment]
Hi @johnantoni ,
Very cool project!
Would you mind commenting what you mean by "social impact"? Is it, for example, how far your posts/tweets get, or is it how interne activity gets tracked?

**Comment by johnantoni** [comment]
Thanks @amchagas i'm thinking on the lines of how your internet activity gets tracked, as that's probably a good place to start.
I'd like to do posts/tweets/facebook but that would require some kind of server, which i can do later on.

**Comment by eBildungslabor** [comment]
Very interesting tool, which would be great to use within my project. So I am looking forward to the realization of the idea. Good luck!

**Comment by eebigger** [comment]
Hi @johnantoni , great project! Could you comment if you would be using data visualizations to create these social impact maps? I really like this idea of mapping impacts on the web!

**Comment by johnantoni** [comment]
> Hi @johnantoni , great project! Could you comment if you would be using data visualizations to create these social impact maps? I really like this idea of mapping impacts on the web!

yes, i'm thinking on the lines of using a google map to visualize the data

**Comment by johnantoni** [comment]
Open Canvas Link
https://docs.google.com/presentation/d/1QHtmGlTXhagjBSPkQidzd766c2ksnP0UAMvLU57wHFE/edit?usp=sharing

**Comment by dajanamulaj** [comment]
Hi @johnantoni. Congrats on your project. :)

Feedback regarding your Open Canvas:
- **Key Metrics**: I think that this part is important for measuring success and I would suggest adding some numbers if possible. For example you could include the number of people you plan to influence. 
- **Unique Value Proposition**: Maybe you can expand this one a bit. For example you could add the lack of such initiatives/platforms or a specific feature of your project which makes it different from other similar projects.  
- **Contributor Channels**: LinkedIn is a great resource to find people who can help you as you can use LinkedIn's features to search with filters for professionals living in different parts of the world.

**Comment by seandiggity** [comment]
Your canvas is great!  Can't wait to see what materializes (and recommend it in our privacy workshops).

A comment on scope: I think it's best to have a limited scope and confine the "footprint" to social media intermediation (e.g. twitter and facebook code/widgets), dodgy javascript trackers, and so on.  Some food for thought:

DuckDuckGo's Privacy Essentials browser addon:
https://addons.mozilla.org/en-US/firefox/addon/duckduckgo-for-firefox/

LibreJS:
https://www.gnu.org/software/librejs/

**Comment by johnantoni** [comment]
@seandiggity totally agree, it would be awesome to do everything but by doing that I'd probably not complete anything. Best to rein the scope in and have it more finite and achievable like you say.

**Comment by johnantoni** [comment]
@dajanamulaj Excellent! thanks so much for the feedback, going to refine it tomorrow so will add these. thanks again.

**Comment by johnantoni** [comment]
Ok, updated. I'll use this to flesh out my readme.

**Comment by johnantoni** [comment]
Readme https://github.com/johnantoni/open-footprint/blob/master/README.md
